modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+
+[id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+compute: <2>
+- hyperthreading: Enabled <3>
+  name: worker
+  replicas: 3
+  platform:
+    vsphere: <4>
+      cpus: 2
+      coresPerSocket: 2
+      memoryMB: 8196
+      osDisk:
+        diskSizeGB: 120
+controlPlane: <2>
+  hyperthreading: Enabled <3>
+  name: master
+  replicas: 3
+  platform:
+    vsphere: <4>
+      cpus: 4
+      coresPerSocket: 2
+      memoryMB: 16384
+      osDisk:
+        diskSizeGB: 120
+metadata:
+  name: cluster <5>
+platform:
+  vsphere:
+    vcenter: your.vcenter.server
+    username: username
+    password: password
+    datacenter: datacenter
+    defaultDatastore: datastore
+    folder: folder
+    network: VM_Network
+    cluster: cluster_name
```

Comment:
Here the cluster_name refer to the name of cluster on vcenter. In order to not confuse with cluster name for ocp cluster <5>, it's better to append "vcenter" keyword for "cluster_name", what do you think?